### ISSUE TITLE
WhatsApp Changed Code?

### ISSUE THREAD

**Issue by dragoj888** [pedroslopez/whatsapp-web.js#1018]
My program works for the past few weeks, but suddenly stopped working, error as follows.
Maybe WhatsApp changed their codes so Puppeteer cannot find them?

```
(node:8988) UnhandledPromiseRejectionWarning: Error: Evaluation failed: TypeError: Cannot read properties of undefined (reading 'default')
    at __puppeteer_evaluation_script__:36:78
    at ExecutionContext._evaluateInternal (/Users/xxx/node_modules/puppeteer/lib/cjs/puppeteer/common/ExecutionContext.js:221:19)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async ExecutionContext.evaluate (/Users/xxx/node_modules/puppeteer/lib/cjs/puppeteer/common/ExecutionContext.js:110:16)
    at async Client.initialize (/Users/xxx/node_modules/whatsapp-web.js/src/Client.js:172:9)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:8988) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:8988) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

```

**Comment by ebta** [comment]
Already using the latest version ? there are some code change since WhatsApp Web v2.2146.9, try to update to the latest version (v1.15.2)

**Comment by mejiafabiandj** [comment]
Same problem, and yes we're using V1.15.2

![image](https://user-images.githubusercontent.com/3708192/145124428-916ffc4c-e804-41fd-856f-f1f625b9ef79.png)

**Comment by irkhamissimo** [comment]
same problem here since yesterday

**Comment by JoseHM8A** [comment]
Update your whatsapp-web.js version to 1.15.3

**Comment by andres-flores-espol** [comment]
Solved using v1.15.3

**Comment by PurpShell** [comment]
solved, then close :D
